package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A platform to make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "nyc --reporter=html --reporter=text mocha  --require  @babel/register  src/tests/*.js --exit"
```

Comment:
How do I confirm that this works?

Reply:
Thank you for the feedback. i have run `npm test` after getting your feedback and got an error because i only had  `@babel/register` without `@babel/core`. Now i have installed `@babel/core` and now things are working well. The following is the screenshot for the output. It is also attached into the PR Screenshot area. Thank you!!!
<img width="1434" alt="Screen Shot 2019-11-29 at 15 25 10" src="https://user-images.githubusercontent.com/58309608/69872973-b3093500-12bf-11ea-9ec7-e0aee3185420.png">

Reply:
Can you set up a dummy test instead?

Reply:
Good evening @adesege , I have setup dummy test. Thank you for the feedback